scripts/install-forms-flow.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+read -p "Please enter the domain name (ex: apps.bronze.aot-technologies.com):" DOMAIN_NAME
+read -p "Please enter the namespace (ex: devin):" NAMESPACE
+
+echo "Installing forms flow to Openshift ..."
+
```

Comment:
Order of templates should be like in this doc : https://aot-technologies.github.io/forms-flow-ai-doc/#kubernetes_installation
Else it will fail on first attempt